### ISSUE TITLE
Jump definition does not work for function call from contract object

### ISSUE THREAD

**Issue by polarker** [alephium/ralph-lsp#340]
In the following code, I can't jump to the `bar` function for contract `foo`
```
let foo = Foo(fooId)
foo.bar()
```

**Comment by polarker** [comment]
This does not work either:
```
let foo = getFoo()
foo.bar()
```

**Comment by polarker** [comment]
Sorry, the contracts I am reviewing has compilation issues. I only notice that once I run `npx run compile`.

Is it possible to show compiler errors to users more explicitly?

**Comment by simerplaha** [comment]
> Is it possible to show compiler errors to users more explicitly?

All the errors returned by invoking `ralphc` during parsing and compilation should already be reported explicitly.

Can we please have an example of a case where the compiler error is not being explicit? This will help us recreating it.

**Comment by polarker** [comment]
Yes, it's reported, but not so easy to notice them. How should I check the errors?

**Comment by simerplaha** [comment]
Oh they should all be reported as regular error diagnostics. So they are highlighted red and reported in "Problems" tab. The errors reported are only the ones coming from Node's API [here](https://github.com/alephium/ralph-lsp/blob/master/compiler-access/src/main/scala/org/alephium/ralph/lsp/access/compiler/RalphCompilerAccess.scala#L66). `npx run compile` is not executed by ralph-lsp.

**Comment by simerplaha** [comment]
Is this issue resolved?

**Comment by polarker** [comment]
I don't have time to test. If it's resolved, let's close it.

**Comment by simerplaha** [comment]
Thanks. I'm closing this as it doesn’t require any changes.
